lib/grp.gd

```diff
@@ -784,6 +784,8 @@ InstallFactorMaintenance( IsSolvableGroup,
 InstallTrueMethod( IsSolvableGroup, IsMonomialGroup );
 InstallTrueMethod( IsSolvableGroup, IsSupersolvableGroup );
 
+InstallTrueMethod( HasIsPerfectGroup, IsGroup and IsSolvableGroup and IsNonTrivial );
```

Comment:
Why is this in the third commit, which is about changing "testinstall" according to its summary? Something seems to be off with how these commits are split.

Reply:
See remark below about making all a single commit.